1.8/administration/theme-problems.md

```diff
@@ -0,0 +1,14 @@
+# Broken themes
+If your theme doesn't look right or is completely broken you can try the following things to help you fix it.
+
+## Check your Board URL
+If your board URL is not set correctly then the paths to the resources in your theme will be broken. To check your board URL go to *ACP > Configuration > General Configuration* > edit the *Board URL* field. Make sure you include **http://** in your URL and don't put a trailing slash.
```

Comment:
This should be:
If your board URL is not set correctly then the paths to the resources in your theme will be broken. To check your board URL go to *ACP > Configuration > Board Settings* > edit the *Board URL* field. Make sure you include **http://** in your URL and don't put a trailing slash.